src/main/java/com/tg5/configuration/ActiveMQConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.tg5.configuration;
+
+import jakarta.jms.ConnectionFactory;
+import org.apache.activemq.ActiveMQConnectionFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.jms.core.JmsTemplate;
+
```

Comment:
Are you sure this is necessary and not just application.yml config lines are needed?